pymc3_hmm/distributions.py

```diff
@@ -432,12 +432,12 @@ def random(self, point=None, size=None):
             Gamma = np.broadcast_to(Gamma, tuple(states.shape) + Gamma.shape[-2:])
 
             # Slices across each independent/replication dimension
-            slices = [slice(None, d) for d in state_shape]
-            slices = tuple(np.ogrid[slices])
+            slice_list = [slice(None, d) for d in state_shape]
+            slices_tuple = tuple(np.ogrid[slice_list])
```

Comment:
Couldn't we just combine these two into one expression and avoid the rename and extra variable?